cmd/root.go

```diff
@@ -0,0 +1,55 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var cfgFile string
+
+var rootCmd = &cobra.Command{
+	Use:   "guppy",
+	Short: "Interact with the Storacha Network",
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	err := rootCmd.Execute()
+	if err != nil {
+		os.Exit(1)
+	}
+}
+
+func init() {
+	cobra.OnInitialize(initConfig)
+
+	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.guppy.yaml)")
+}
+
+// initConfig reads in config file and ENV variables if set.
+func initConfig() {
```

Comment:
To simplify things I'd consider introducing configuration (files) in a follow on. Using a mixture of vipers different binding methods, which work with cobra: https://github.com/spf13/viper?tab=readme-ov-file#working-with-flags